org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -248,6 +255,8 @@ private void startConnection() throws IOException {
 	 */
 	@Override
 	public void stop(BundleContext bundleContext) throws Exception {
+		GradleBuildSupport.stop();
+		ResourcesPlugin.getWorkspace().save(true, null);
```

Comment:
Should we do this here or at `JDTLanguageServer.shutdown()`?

Reply:
I have update the PR.